packages/react/src/Grid/Grid.tsx

```diff
@@ -99,6 +99,41 @@ export const Grid = forwardRef(
   ),
 ) as GridComponent
 
+type GridCellFullWidthProp = {
+  /** Whether the cell spans the full width of the grid. */
+  fullWidth?: boolean
+  span?: never
+  start?: never
+}
+
+type GridCellColumnProps = {
+  fullWidth?: never
+  /** The amount of grid columns the cell spans. */
+  span?: GridColumnNumber | GridColumnNumbers
+  /** The index of the grid column the cell starts at. */
+  start?: GridColumnNumber | GridColumnNumbers
+}
+
+export type GridCellProps = (GridCellFullWidthProp | GridCellColumnProps) &
+  PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+
+const GridCell = forwardRef(
```

Comment:
I thought we wanted to keep subcomponents in seperate files? Similar to `Accordion`

Reply:
Yeah, we discussed this earlier. I wrote down:

> 1 story, 1 css file, seperate files for child components, with their own test files. We do not directly expose child components to consumers.

If we agree on that convention, we should document it somewhere.

It's not a problem for the exports. If you try to import `Accordion*` now, you only get `Accordion`, `AccordionProps` and `AccordionSectionProps`, ie only the things specified in `index.ts` of the component.

Reply:
Yes, pushed.